assets/sass/components/_FrontPageContainer.scss

```diff
@@ -0,0 +1,51 @@
+.FrontPageContainer {
+  margin: 0;
+  width: 100vw;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  padding: 6em 0;
+  text-align: center;
```

Comment:
I do not like the centered text, as it is harder to read and does not look good.